evap/static/ts/src/datagrid.ts

```diff
@@ -253,7 +264,8 @@ export class TableGrid extends DataGrid {
     }
 
     protected findSearchableCells(row: HTMLElement): HTMLElement[] {
-        return [...row.children] as HTMLElement[];
+        const children = [...row.children] as HTMLElement[];
+        return this.searchableColumnIndices.map(index => children[index]).filter(cell => cell !== undefined);
```

Comment:
To remove any type casts, you can do
```suggestion
        return this.searchableColumnIndices.map(index => {
            const child = row.children[index];
            assert(child instanceof HTMLElement);
            return child;
        });
```
after importing assert from our utils.